mappings/net/minecraft/world/World.mapping

```diff
@@ -232,6 +233,20 @@ CLASS net/minecraft/unmapped/C_cdctfzbn net/minecraft/world/World
 	METHOD m_offwleab getSkyAngleRadians (F)F
 		ARG 1 tickDelta
 	METHOD m_olkmteoq getMapState (Lnet/minecraft/unmapped/C_qqxyyzzm;)Lnet/minecraft/unmapped/C_nvpllgmg;
+	METHOD m_onifczix createExplosion (Lnet/minecraft/unmapped/C_astfners;Lnet/minecraft/unmapped/C_sbxfkpyv;Lnet/minecraft/unmapped/C_dmexlawk;DDDFZLnet/minecraft/unmapped/C_cdctfzbn$C_dlzvrbnu;ZLnet/minecraft/unmapped/C_nqucohct;Lnet/minecraft/unmapped/C_nqucohct;Lnet/minecraft/unmapped/C_cjzoxshv;)Lnet/minecraft/unmapped/C_aahhrzpf;
+		ARG 1 entity
```

Comment:
This could use a more descriptive name, maybe
1) `immedateCausingEntity`
0) `immediateCauser` (would also mean updating `getCausingEntity`)
0) `immediateIgniter` (same)
0) `parent` (probably too general, but maybe inspires better ideas)